src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest_getHousingLocation.kt

```diff
@@ -50,6 +55,9 @@ class OffenderResourceIntTest_getHousingLocation : ResourceTest() {
       .responseBody.blockFirst()!!
 
     assertThat(location).isEqualTo(OffenderLocation())
```

Comment:
This test was interferring with the refactored Kotlin (from Java) test so needed tidying up code adding